examples/deeplab/README.md

```diff
@@ -0,0 +1,36 @@
+# Examples of DeepLab
+
+## Performance
+
+| Model | Reference | ChainerCV (weight conversion) |
+|:-:|:-:|:-:|
+| Pascal VOC (single scale) |  82.36 % * |  82.36 % |
+| Cityscapes (single scale) | 79.12 % * | 79.14 % |
+
+Scores are measured by mean Intersection over Union (mIoU).
+\*: Although the official repository reports a score of multi-scale prediciton, public pretrained graph is for single-scale prediction.
+So we evaluated the pretrained graph using `eval_semantic_segmentation` in ChainerCV.
+
+## Demo
+This demo downloads Cityscapes pretrained model automatically if a pretrained model path is not given.
```

Comment:
This is currently not true. Could you set the default value of `pretrained_model` to `cityscapes`?